docs/Advanced-Usage/Debugging-and-Profiling-on-FPGA/Printf-Synthesis.rst

```diff
@@ -51,15 +51,18 @@ Runtime Arguments
 ---------------------------
 
 **+print-file**
-    Specifies the file into which the synthesized printf log should written.
+    Specifies the file name stem. Generated files will be of the form `<print-file><N>`,
```

Comment:
Is "stem" the right term here? I've never heard it being used in this context

Reply:
I would say "prefix" instead of stem

Reply:
Sounds good.